Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -416,7 +416,6 @@ extension PlaceCarouselViewController: PlacesProviderDelegate {
     }
 
     func placesProvider(_ controller: PlacesProvider, didError error: Error) {
-        showErrorMessageIfNoPlaces()
```

Comment:
Initially I was a bit confused as to what this network call was doing but it looks like we actually don't care about if it error'ed or succeeded since it's just a ping. We could probably just remove the `didError` protocol method altogether. The only 'error' state we have for fetching places should be handled by `didTimeout`.